app/models/course/material/folder.rb

```diff
@@ -10,9 +12,17 @@ class Course::Material::Folder < ActiveRecord::Base
   # TODO: Remove this after schema_validations #21 was closed
   validates :name, uniqueness: { case_sensitive: false, scope: :parent_id }, if: :parent
 
+  def self.after_course_create(course)
+    course.material_folders.create(name: 'Root') unless course.root_folder
```

Comment:
Oops. I think it's not needed anymore, it's for solving the validation issue before.